gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -263,6 +263,34 @@ case class TransformPreOverrides(isAdaptiveContextOrTopParentExchange: Boolean)
         genFilterExec(plan)
       case plan: HashAggregateExec =>
         genHashAggregateExec(plan)
+      case plan: SortAggregateExec =>
```

Comment:
The following case could be possible
```
               +- ColumnarExchange (257)
                  +- ^ CHHashAggregateExecTransformer (255)
                     +- ColumnarExchange (254)
                        +- ^ CHHashAggregateExecTransformer (252)
                           +- ^ FilterExecTransformer (251)
                              +- ^ ExpandExecTransformer (250)
                                 +- ^ ProjectExecTransformer (249)
                                    +- ^ CHShuffledHashJoinExecTransformer Inner (248)
                                       :- RowToCHNativeColumnar (238)
                                       :  +- SortAggregate (237)
                                       :     +- SortAggregate (236)
                                       :        +- CHNativeColumnarToRow (235)
                                       :           +- ^ SortExecTransformer (233)
                                       :              +- ^ ProjectExecTransformer (232)
                                       :                 +- ^ CHShuffledHashJoinExecTransformer LeftOuter (231)
                                       :                    :- ReusedExchange (229)
                                       :                    +- ReusedExchange (230)
```
One `SortAggExec`  is followed by the other directly.

Reply:
It looks there is no difference for consecutive sort agg in the replacement. Right? Am I missing something?

Reply:
The pattern doesn't match this case, sort agg will not be replaced

Reply:
@lgbo-ustc, it looks we don't limit the child to be sort op in the replacement.
Did you enable the proposed replacement for CH backend? Currently, we differentiate the enabling for different backends. For CH backend, we disabled it in this PR. If you would like to test on CH, you need to apply the below change.
```
diff --git a/backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHBackend.scala b/backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHBackend.scala
index 1973e4e5..7f696858 100644
--- a/backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHBackend.scala
+++ b/backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHBackend.scala
@@ -69,6 +69,8 @@ object CHBackendSettings extends BackendSettings with Logging {
       ".customized.buffer.size"
   val GLUTEN_CLICKHOUSE_CUSTOMIZED_BUFFER_SIZE_DEFAULT = "4096"
 
+  override def replaceSortAggWithHashAgg: Boolean = GlutenConfig.getConf.forceToUseHashAgg
+
   override def supportFileFormatRead(
       format: ReadFileFormat,
       fields: Array[StructField],
```